Big Shoulders Inline: Version 2.002 added

yanone opened the pull request:
Taken from the upstream repo https://github.com/xotypeco/big_shoulders at commit https://github.com/xotypeco/big_shoulders/commit/2a671cfac468a342756b3c37a32845403602a214.

Resolves #7830
## PR Checklist:

- [ ] `minisite_url` definition in the METADATA.pb file for commissioned projects
- [ ] `tags` are added for NEW FONTS
- [ ] `primary_script` definition in the METADATA.pb file for all projects that have a primary non-Latin based language support target
- [ ] `subsets` definitions in the METADATA.pb reflect the actual subsets and languages present in the font files (in alphabetic order). For **CJK fonts**, only include one of the following subsets `chinese-hongkong`, `chinese-simplified`, `chinese-traditional`, `korean`, `japanese`.
- [ ] Fontbakery checks are reviewed and failing checks are resolved in collaboration with the upstream font development team
- [ ] Diffenator2 regression checks for revisions on all projects that are currently in production
- [ ] Designers bio info have to be present in the designer catalog (at least an issue should be opened for tracking this, if they are not)
- [ ] Check designers order in metadata.pb, since the first one of the list appears as “principal designer”
- [ ] Social media formatted visual assets for all new commissioned projects in the Drive directory, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet
- [ ] Social media content draft for all new commissioned projects in the Drive directory and Social Media tracker spreadsheet, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet

yanone commented:
`Big Shoulder Inline Text` and `Big Shoulder Inline Display` need to be hidden/discontinued when this goes live.

emmamarichal commented:
Interpolations issues:
[BigShouldersInline[opsz,wght].pdf](https://github.com/user-attachments/files/18688015/BigShouldersInline.opsz.wght.pdf)

 + see my other comments in the Big Shoulders Stencil PR

yanone commented:
Updated

yanone commented:
I fixed the vertical metrics, interpolation issues (except `kgreenlandic` which I believe to be a false positive, I checked it) and added `Eturned` and added the `ss01` description. The other WARNs I’m ignoring.
